### DISCUSSION TITLE
bot咋做的？

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#2942] [提问] [answered]
就挺好奇这bot要咋搞一个~虽然我用不上~

**Comment by MoYuan-CN** [comment]
猜过去 Github Api + Webhook 然后检测 hook 类型

**Comment by WorldHim** [comment, marked as answer]
#894

> github 提供了 API，你可以翻翻文档……Graph API 和 Rest API（
